infrastructure/ansible/roles/traffic_ops/templates/cdn.conf.j2

```diff
@@ -15,17 +15,6 @@
    "cors" : {
       "access_control_allow_origin" : "{{ to_cors_access_control_allow_origin }}"
    },
-   "hypnotoad" : {
-      "accept_interval" : {{ to_hypnotoad_accept_interval }},
-      "group" : "{{ to_group }}",
-      "heartbeat_timeout" : {{ to_hypnotoad_heartbeat_timeout }},
-      "listen" : [
-         "https://[::]:{{ to_hypnotoad_port }}?cert={{ to_certs_cert }}&key={{ to_certs_key }}&ca={{ to_certs_ca }}&verify={{ to_certs_verify}}&ciphers={{ to_hypnotoad_ciphers | join(":") }}"
-      ],
-      "pid_file" : "{{ to_hypnotoad_pidfile }}",
-      "user" : "{{ to_user }}",
-      "workers" : {{ to_hypnotoad_number_of_workers }}
-   },
```

Comment:
With this section removed, these variables are unused and should be removed:

https://github.com/apache/trafficcontrol/blob/c60b07bddcdd6c82697660ab87b41ef888f3bef4/infrastructure/ansible/roles/traffic_ops/defaults/main.yml#L49-L50

https://github.com/apache/trafficcontrol/blob/c60b07bddcdd6c82697660ab87b41ef888f3bef4/infrastructure/ansible/roles/traffic_ops/defaults/main.yml#L67-L78